src/components/views/rooms/wysiwyg_composer/hooks/usePlainTextInitialization.ts

```diff
@@ -19,7 +19,7 @@ import { RefObject, useEffect } from "react";
 export function usePlainTextInitialization(initialContent = "", ref: RefObject<HTMLElement>): void {
     useEffect(() => {
         if (ref.current) {
-            ref.current.innerText = initialContent;
+            ref.current.innerHTML = initialContent;
```

Comment:
We read from this composer using `.innerHTML`. Setting using `.innerHTML` makes more sense for consistency as it prevents linebreaks switching back and forth between `\n` and `<br>`, for instance.

Reply:
I think it would be worth turning this into a code comment.